doc/policy/packages.md

```diff
@@ -117,3 +118,30 @@ backward-compatible for users and applications that rely on p2p transaction rela
 package validation should not prevent the acceptance of a transaction that would otherwise be
 policy-valid on its own. By always accepting a transaction that passes individual validation before
 trying package validation, we prevent any unintentional restriction of policy.
+
+### Package Replace by Fee
+
+Child-with-unconfirmed-parents packages may replace mempool transactions under the following conditions:
+
+1. The child transaction does not conflict with any transactions in the mempool.
```

Comment:
Yes you're right. Thanks for the feedback and clear explanation, adding some similar language to the doc!

Reply:
> This rule startled me a bit at first, it may be worth mentioning that if you want to update the package's child, you should do a normal RBF of just that child.
> 
> 
> 
> What you mean is that package-rbf applies for the case where we want for example to replace `{commitTxAlice, anchorAlice}` by `{commitTxBob, anchorBob}` where `commitTxAlice` and `commitTxBob` conflict, but we cannot use package-rbf to replace `{commitTxAlice, anchorAlice1}` by `{commitTxAlice, anchorAlice2}` where `anchorAlice2` pays more fees than `anchorAlice1`. In that case we should submit `anchorAlice2` alone and it will go through the usual RBF rules. Is that correct?
> 
> 
> 
> Also you don't want to mix up package-rbf and normal rbf, so a child tx cannot create conflicts "outside of the package" IIUC.

Reply:
> This rule startled me a bit at first, it may be worth mentioning that if you want to update the package's child, you should do a normal RBF of just that child.
> 
> 
> 
> What you mean is that package-rbf applies for the case where we want for example to replace `{commitTxAlice, anchorAlice}` by `{commitTxBob, anchorBob}` where `commitTxAlice` and `commitTxBob` conflict, but we cannot use package-rbf to replace `{commitTxAlice, anchorAlice1}` by `{commitTxAlice, anchorAlice2}` where `anchorAlice2` pays more fees than `anchorAlice1`. In that case we should submit `anchorAlice2` alone and it will go through the usual RBF rules. Is that correct?
> 
> 
> 
> Also you don't want to mix up package-rbf and normal rbf, so a child tx cannot create conflicts "outside of the package" IIUC.

Reply:
> This rule startled me a bit at first, it may be worth mentioning that if you want to update the package's child, you should do a normal RBF of just that child.
> 
> 
> 
> What you mean is that package-rbf applies for the case where we want for example to replace `{commitTxAlice, anchorAlice}` by `{commitTxBob, anchorBob}` where `commitTxAlice` and `commitTxBob` conflict, but we cannot use package-rbf to replace `{commitTxAlice, anchorAlice1}` by `{commitTxAlice, anchorAlice2}` where `anchorAlice2` pays more fees than `anchorAlice1`. In that case we should submit `anchorAlice2` alone and it will go through the usual RBF rules. Is that correct?
> 
> 
> 
> Also you don't want to mix up package-rbf and normal rbf, so a child tx cannot create conflicts "outside of the package" IIUC.